docs/manual/docs/cmd/spo/app/app-retract.md

```diff
@@ -14,7 +14,8 @@ Option|Description
 ------|-----------
 `--help`|output usage information
 `-i, --id <id>`|ID of the app to retract. Needs to be available in the tenant app catalog.
-`-u, --appCatalogUrl [appCatalogUrl]`|(optional) URL of the tenant app catalog site. If not specified, the CLI will try to resolve it automatically
+`-u, --appCatalogUrl [appCatalogUrl]`|URL of the tenant or site app catalog. It must be specified when the scope is `sitecollection`
+`-s, --scope [scope]`|Scope of the app catalog: tenant|sitecollection. Default tenant
```

Comment:
`tenant|sitecollection` should be wrapped in backticks. Currently, the pipe creates a new table column and some of the information is cut off.